src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/ViewBuffer.cs

```diff
@@ -90,55 +91,66 @@ public int Count
         }
 
         /// <inheritdoc />
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
It would be good to capture some notes (in a comment) about the analysis that leads to the particular optimizations that were done here. These kinds of things are easy to break

Reply:
Added comments